### ISSUE TITLE
[Suggestion] new way to find bases with servers that have anti cheat

### ISSUE THREAD

**Issue by yalolo1** [MeteorDevelopment/meteor-client#4909]
### Describe the feature

 hey i think u should add some esp thats show u holes that got mined straight down that can rly rly help alot in finding bases on any server with anti cheat cus a lot of ppl just mine straight down and put like 2 blocks so u wont see the way to the base ty hope yall add it :)

### Before submitting a suggestion

- [X] This feature doesn't already exist in the client. (I have checked every module and their settings on the **latest dev build**)

- [X] This wasn't already suggested. (I have searched suggestions on GitHub and read the FAQ)

**Comment by benjith17** [comment]
⬆️

**Comment by Kolotheegg** [comment]
Tunnel ESP?

**Comment by ahykono** [comment]
> Tunnel ESP?

Yeah that would be tunnel ESP (It does exist) however it only shows horizontal and not vertical tunnels
I guess it could be added as a right click setting
